src/python/WMCore/WMSpec/StdSpecs/PromptReco.py

```diff
@@ -61,7 +61,7 @@ def buildWorkload(self):
                                                splitAlgo = self.procJobSplitAlgo,
                                                splitArgs = self.procJobSplitArgs,
                                                stepType = cmsswStepType,
-                                               forceUnmerged = True)
+                                               forceUnmerged = ["write_ALCARECO"] if 'ALCARECO' in self.writeTiers else False)
```

Comment:
I don't want to do that since the parameter is used elsewhere as just a boolean, these other places don't care about the tier functionality, they just want to turn this off period. This is a general parameter to force all output to unmerged (same as forceMerged is a general parameter to force all output to merged). All I want is to support that forcing to unmerged to be partial in this one case, not general. I do not want every other use of this parameter to be exposed to this list option, they should be able to keep using the boolean syntax.